lib/core/src/Cardano/Wallet.hs

```diff
@@ -637,6 +639,8 @@ newWalletLayer tracer block0 feePolicy db nw tl = do
         let utxo = availableUTxO @s @t w
         (sel, _utxo') <- withExceptT ErrEstimateTxFeeCoinSelection $
             CoinSelection.random opts recipients utxo
+        _ <- withExceptT ErrEstimateTxFeeBackend $
```

Comment:
There's no need for `_ <-` if the function returns `()`. It also makes it obvious that we aren't actually discarding anything.